ui/lib/kubernetes/addon/templates/configuration.hbs

```diff
@@ -1 +1 @@
-<Page::Configuration @model={{this.model}} />
\ No newline at end of file
+<Page::Configuration @config={{this.model.config}} @backend={{this.model.backend}} />
```

Comment:
I'm a big fan of this pattern! It makes it easier to access these properties in the component template :)